index.js

```diff
@@ -142,7 +141,7 @@ File.prototype.inspect = function() {
 };
 
 File.isVinyl = function(file) {
-  return (file && file._isVinyl === true) || false;
+  return Boolean(file && file._isVinyl);
```

Comment:
This check was actually changed rather recently and I don't see a reason to change it.  Also, if someone set their own `_isVinyl = 'foo'` the above change would report true when it should report false.